src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt

```diff
@@ -125,11 +126,10 @@ class RecipeMarkdownGenerator : Runnable {
         if (recipeSources.isNotEmpty()) {
             recipeOrigins = RecipeOrigin.parse(recipeSources)
 
-            // Write latest-versions-of-every-openrewrite-module.md
-            createLatestVersionsJs(outputPath, recipeOrigins)
-            createLatestVersionsMarkdown(outputPath, recipeOrigins)
-
             if (recipeClasspath.isEmpty()) {
+                // Write latest-versions-of-every-openrewrite-module.md for the base recipes only
+                createLatestVersionsJs(outputPath, recipeOrigins)
+                createLatestVersionsMarkdown(outputPath, recipeOrigins)
                 return
             }
```

Comment:
What do you mean by base recipes only here?

For context this much quicker mode was added specifically such that we can have a nightly job update the latest version page in the docs:
https://github.com/openrewrite/rewrite-docs/blob/eb35e21e6bc80ff5858176e7ecf4bc8809fd3746/.github/workflows/update-docs.yml#L31-L33

If this only shows a subset of recipe modules, then that's no good, as the page would miss a lot of recipe modules.

Reply:
will remove, was present before.